src/Cli/dotnet/Commands/Tool/Update/ToolUpdateCommand.cs

```diff
@@ -78,6 +78,15 @@ internal static void EnsureEitherUpdateAllOrUpdateOption(
         }
     }
 
+    internal static void EnsureNoConflictPackageIdentityVersionOption(ParseResult parseResult)
```

Comment:
Wondering if you could put this somewhere else and share

Reply:
Was having some trouble with that because of the way I would have to call it, aka, passing in the argument. 
I will do a followup pr seeing if this argument can be implemented in other places in the cli. If that is the case, I will see if we can figure out a way to reuse this bit